File: app/src/main/java/ua/ck/zabochen/englishverbs/utils/Tools.java
package ua.ck.zabochen.englishverbs.utils;

import android.content.Context;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.support.v4.graphics.drawable.RoundedBitmapDrawable;
import android.support.v4.graphics.drawable.RoundedBitmapDrawableFactory;

import java.io.IOException;

public class Tools {

    public static RoundedBitmapDrawable getRoundedImageFromAssets(Context context, String assetImageUrl) {

        RoundedBitmapDrawable roundedBitmapImage = null;

        try {

            // Source image
            Bitmap sourceBitmapImage = BitmapFactory.decodeStream(context.getAssets().open(assetImageUrl));

            // Crop center image
            Bitmap editBitmapImage = Bitmap.createBitmap(
                    sourceBitmapImage,
                    sourceBitmapImage.getWidth() / 2 - sourceBitmapImage.getHeight() / 2,
                    0,
                    sourceBitmapImage.getHeight(),
                    sourceBitmapImage.getHeight()
            );

            // Rounded image
            roundedBitmapImage = RoundedBitmapDrawableFactory.create(
                    context.getResources(),
                    editBitmapImage
            );
            roundedBitmapImage.setCornerRadius(Math.max(editBitmapImage.getWidth(), editBitmapImage.getHeight() / 2.0f));

        } catch (IOException e) {
            e.printStackTrace();
        }

        return roundedBitmapImage;

    }

    public static RoundedBitmapDrawable getRoundedImageFromResources(Context context, int imageId) {

        // Source image
        Bitmap sourceBitmapImage = BitmapFactory.decodeResource(
                context.getResources(),
                imageId
        );

        // Crop center image
        Bitmap editBitmapImage = Bitmap.createBitmap(
                sourceBitmapImage,
                sourceBitmapImage.getWidth() / 2 - sourceBitmapImage.getHeight() / 2,
                0,
                sourceBitmapImage.getHeight(),
                sourceBitmapImage.getHeight()
        );

        // Rounded image
        RoundedBitmapDrawable roundedBitmapImage = RoundedBitmapDrawableFactory.create(
                context.getResources(),
                editBitmapImage
        );
        roundedBitmapImage.setCornerRadius(Math.max(editBitmapImage.getWidth(), editBitmapImage.getHeight() / 2.0f));

        return roundedBitmapImage;

    }
}

File: app/src/main/java/ua/ck/zabochen/englishverbs/utils/Tools.kt
package ua.ck.zabochen.englishverbs.utils

import android.content.Context
import android.graphics.Bitmap
import android.graphics.BitmapFactory
import android.widget.Toast
import androidx.core.graphics.drawable.RoundedBitmapDrawable
import androidx.core.graphics.drawable.RoundedBitmapDrawableFactory

class Tools {

    companion object {

        fun bitmapImageFromAssets(context: Context, imagePath: String): Bitmap {
            return BitmapFactory.decodeStream(context.assets.open(imagePath))
        }

        fun roundedImageFromAssets(context: Context, imagePath: String): RoundedBitmapDrawable {

            // Source image
            val sourceBitmapImage: Bitmap = BitmapFactory.decodeStream(context.assets.open(imagePath))

            // Crop center image
            val editBitmapImage: Bitmap = Bitmap.createBitmap(
                    sourceBitmapImage,
                    sourceBitmapImage.width / 2 - sourceBitmapImage.height / 2,
                    0,
                    sourceBitmapImage.height,
                    sourceBitmapImage.height
            )

            // Rounded image
            val roundedBitmapImage: RoundedBitmapDrawable = RoundedBitmapDrawableFactory.create(
                    context.resources,
                    editBitmapImage
            )
            roundedBitmapImage.cornerRadius = Math.max(editBitmapImage.width.toFloat(), editBitmapImage.height / 2.0f)
            return roundedBitmapImage
        }

    }

}

// Extensions

fun Context.showToast(message: String, duration: Int = Toast.LENGTH_LONG) {
    Toast.makeText(this, message, duration).show()
}